renderer/parameters/parameters.go

```diff
@@ -21,7 +21,7 @@ const (
 
 var (
 	// VarArgRegexp defines the extra variable parameter format
-	VarArgRegexp = matcher.Must(`^(?P<name>\S+)=(?P<value>[\S ]*)$`)
+	VarArgRegexp = matcher.Must(`^(?P<name>[^=]+)=(?P<value>[\S ]*)$`)
```

Comment:
`\S` is equivalent to `[^ \t\r\n\f]` so to add `=` you'd get `[^ \t\r\n\f=]` but adding a unit test to make sure would be ideal

Reply:
https://github.com/VirtusLab/render/pull/45/files